utils/catatonit/Makefile

```diff
@@ -0,0 +1,43 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=catatonit
+PKG_VERSION:=0.1.7
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/openSUSE/catatonit.git
+PKG_SOURCE_DATE:=2022-03-07
+PKG_SOURCE_VERSION:=d8d72fea155c144ed3bf298a35a1aba5625a5656
+PKG_MIRROR_HASH:=skip
+
+PKG_BUILD_PARALLEL:=1
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+
+PKG_MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE_FILES:=COPYING
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/catatonit
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=A container init that is so simple it's effectively brain-dead.
+  URL:=https://github.com/openSUSE/catatonit
+  DEPENDS:=
```

Comment:
It's weird, but there are no dependencies for catatonit?

Reply:
AFAIK, no.

Reply:
No, there are no dependencies.